R/layers2traces.R

```diff
@@ -622,6 +622,16 @@ to_basic.GeomQuantile <- function(data, prestats_data, layout, params, p, ...){
   dat
 }
 
+#' @export
+to_basic.GeomRectCS <- function(data, ...){
+  to_basic.GeomRect(data, ...)
+}
+#' @export
+to_basic.GeomLinerangeBC <- function(data, ...){
+  data[, c("xend", "y", "yend")] <- data[, c("x", "ymin", "ymax")]
+  to_basic.GeomSegment(data, ...)
+}
```

Comment:
We should probably do more to come up with better `hovertext`:

Reply:
I'll look into this